analytics.js

```diff
@@ -19,19 +19,117 @@
         /* eslint-enable */
     }
 
-    Analytics.prototype.sendEvent = function(action, data) {
-        // empty label if missing value for it and add the value,
-        // the value should be integer or null
-        let value = data.value;
-
-        value = value ? Math.round(parseFloat(value)) : null;
-        const label = data.label || '';
-
-        // Intentionally use string concatenation as analytics needs to work on
-        // IE but this file does not go through babel.
-        // eslint-disable-next-line prefer-template
-        ga('send', 'event', 'jit.si', action + '.' + data.browserName,
-            label, value);
+    /**
+     * Extracts the integer to use for a Google Analytics event's value field
```

Comment:
i was confused by there being GA logic in this general file, but then lenny just told me this is a google analytics-specific file, is that right? maybe we should name it that way since `analytics.js` sounds generic?

Reply:
Agreed. I'll see if it's an easy change.